tests/frameworks/test_sanic.py

```diff
@@ -92,7 +92,7 @@ def test_404(self):
         self.assertEqual(result.status_code, 404)
 
         spans = tracer.recorder.queued_spans()
-        self.assertEqual(len(spans), 3)
+        self.assertEqual(len(spans), 4)
```

Comment:
Out of curiosity, what caused this change of behavior?

Reply:
there is an extra log span coming up after the changes, in the error test cases there are some logs and after the async_tracer is also available for the logging instrumentation, the extra span appears here